File: js_basic/fizzBuzz.js
let number1 = parseInt(process.argv[2]);
let number2 = parseInt(process.argv[3]);

for(i = 1; i <= 100; i++){
    if(i % number1 === 0 && i % number2 === 0){
        console.log("fizzbuzz");
    }
    else if(i % number1 === 0){
        console.log("fizz");
    }else if(i % number2 === 0){
        console.log("buzz");
    }else{
        console.log(i);
        }
    }

File: ruby-arrays-and-hashes/fizzbuzz.rb
array = []
for num in 1..100
    if num % 3 == 0 && num % 5 == 0
        array.push("FIZZBUZZ")
    elsif num % 3 == 0
        array.push("FIZZ")
    elsif num % 5 == 0
        array.push("Buzz")
    else 
        array.push(num)
    end
end

print array


